src/views/portal/components/VerticleTimeline.tsx

```diff
@@ -0,0 +1,64 @@
+import ListGroup from "react-bootstrap/ListGroup";
+import styles from "views/portal/Portal.module.scss";
+
+import { Form } from "react-bootstrap";
+
+interface TimelineProps {
+	status: string | null;
+	application_submitted_date: string | null;
+	verdict_submitted_date?: string | null;
+	forms_submitted_date?: string | null;
+}
+
+function VerticleTimeline(props: TimelineProps) {
+	const first_item = (
+		<ListGroup.Item className={styles.list_item}>
+			<Form.Check disabled={true} checked={true} className={styles.check} />
```

Comment:
Using checkboxes may be misleading to the user since the box is checked no matter the status (as long as the stage is completed). This problem is apparent when the checkmark is the same for both accepted and denied in `second_item`. Try using the assets/icons from #72 to represent something being completed or in progress.